scripts/lint-mdc.ts

```diff
@@ -399,12 +397,6 @@ function checkAllIdentifierNames(inputFilePath: string, inputCode: string) {
     ClassMethod: checkAccessibility,
     ClassProperty: checkAccessibility,
   });
-
-  babelTraverse(ast, {
```

Comment:
do we need to remove all of this in this PR?

Reply:
I meant to turn this off temporarily. Good catch, reverting this.